tailwind.config.js

```diff
@@ -41,6 +41,7 @@ module.exports = {
 
     fontSize: {
       // Defaults are: https://tailwindcss.com/docs/font-size
+      "2xs": ["0.625rem", { lineHeight: "1.25rem", letterSpacing: "0.15em" }],
```

Comment:
I don't think we should be adding a new class inside `tailwind.config.js` for this one-off scenario, which will be scarcely used. You could use Tailwind's arbitrary values instead: https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values

Reply:
I'm struggling to get the arbitrary-property syntax correct. I've got it working with vanilla CSS instead, though.
```
  font-size: 0.625rem;
  line-height: 1.25rem;
  letter-spacing: 0.15em;
```

Reply:
I see! We might be using an older version of Tailwind, where this function didn't exist. 

I replaced the `letter-spacing: 0.15em` with the almost identic `tracking-widest` alternative (`0.1em`), `line-height: 1.25rem;` with the `leading-5` Tailwind equivalent, and kept the font size value as vanilla CSS: You can check it [here](https://github.com/streamlit/docs/pull/845/commits/de04c4eb6b90a10404b44d980801fde92538496c). Let me know if that looks good to you!

Reply:
LGTM